python/smashbox/utilities/__init__.py

```diff
@@ -26,11 +24,26 @@ def setup_test():
     If exception is raised then the testcase execution is aborted and smashbox terminates with non-zero exit code,
 
     """
-    reset_owncloud_account(num_test_users=config.oc_number_test_users)
-    reset_rundir()
-    reset_server_log_file()
+    import imp,sys
+    global test_reporter
+    #check prerequisites
+    try:
+        imp.find_module('numpy')
+        imp.find_module('netifaces')
+        imp.find_module('pycurl')
+    except ImportError,e:
+        logger.error(e) 
+        sys.exit()
+    
+    try:
+        import smashbox.test_reporter
+        test_reporter = smashbox.test_reporter.Test_Reporter(os.path.basename(_smash_.args.test_target),config,_smash_.workers,manager)
+    except:
+        pass
+    
+    #initialize the synchronisation client
+    sync_client_start()  
```

Comment:
sync_client_start is reposponsible for execution of these "client start functions" from "setup_test()"